.storybook/themes/news.scss

```diff
@@ -1329,6 +1329,28 @@
 						),
 					),
 				),
+				xl-promo: (
+					components: (
+						heading: (
+							font-family: var(--font-family-primary),
+							font-size: var(--global-font-size-10),
+							font-weight: var(--global-font-weight-7),
+							line-height: 1.3,
```

Comment:
This is Heading Level 3 at Default. Should we have `var(--global-line-height-4)` here instead?
Or are we using this until the global line-height tokens are multipliers?

Reply:
Until the line-height refactor I didn't want to put in something that might be cutting off text so I used the figma values.

Reply:
I used these values and this is sadly the result:
![image](https://user-images.githubusercontent.com/2287238/179977973-7c3dce5d-e854-4537-8e13-fa902bb69428.png)

Reply:
using pixel values